### PR TITLE
registering ishita303

### PR DESCRIPTION
**Pull request by ishita303**
Hey Maintainer, 

You are doing a great job, kindly accept my request.

### CODE REVIEW

File: content/participants/ttjc@ishita303/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+I sing and play guitar. I started blogging recently, also started with a novel.
+
+### help I need?
+
+I would love to learn more about APi's and deploying applications.
+
+### how to contact me?
+
+- Telegram
```
Comment (gkanishk) [inline content/participants/ttjc@ishita303/index.md:32]: can you mention links to your profile

Comment (ishita303) [reply content/participants/ttjc@ishita303/index.md:32]: My portfolio is not live yet, you can view over linkedin- https://www.linkedin.com/in/ishita-jain-459075175/

Comment (gkanishk) [reply content/participants/ttjc@ishita303/index.md:32]: From that, I mean links for contact as you mentioned telegram and LinkedIn.
I will add your LinkedIn profile in the contact me section 😎

Comment (ishita303) [reply content/participants/ttjc@ishita303/index.md:32]: Sure, Linkedin- https://www.linkedin.com/in/ishita-jain-459075175/

**Comment by gkanishk** [comment]
Welcome On-Board!!

**Pull request merged**